tests/unit/btree_stress_test.c

```diff
@@ -255,6 +265,12 @@ CTEST2(btree_stress, test_random_inserts_concurrent)
       (cache *)&data->cc, &data->dbtree_cfg, packed_root_addr, nkvs, data->hid);
    ASSERT_NOT_EQUAL(0, rc, "Invalid ranges in packed tree\n");
 
+   // Exercise print method to verify that it basically continues to work.
+   btree_print_tree(Platform_default_log_handle,
```

Comment:
This call here is a bit out-of-line with the nature of this "stress" test, whose original intent was to do some stress-activity to exercise BTree interfaces.

The reason this is here is because in this test we are invoking `pack_tests()` -> `btree_pack()`, and it is through this interface that the fields in`btree_pivot_stats{}` are updated.

This change-set enhances `btree_print_btree_pivot_stats()` which prints these fields to now also print them using unit-specifiers.

I needed a way to exercise this print code to cross-check that the right values are being printed.

For that purpose, this small change leverages the work done in this BTree stress test to also invoke the print methods as a way to do this validation.

Reply:
As mentioned elsewhere, the outputs from the updated print methods looks like:

```
BTree node at addr=1333133312
{
**  INDEX NODE
**  Header ptr: 0x7f55c36ce000
**  addr: 1333133312
**  next_addr: 0
**  next_extent_addr: 0
[...]
Array of 3 index entries:
[ 0]: key=0x00000d093a9a17000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
   child_addr=1333657600
   (num_kvs=377607
    key_bytes=21716772 (~20.71 MiB)
    message_bytes=260863299 (~248.77 MiB))
[ 1]: key=0x60aaf36fb8dd4d000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
   child_addr=1333661696
   (num_kvs=379751
    key_bytes=21839065 (~20.82 MiB)
    message_bytes=262304591 (~250.15 MiB))

[... And so on ...]

```